src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -178,31 +186,40 @@ private async Task StartDashboardAsync(CancellationToken cancellationToken = def
 
     private async Task WaitForHttpSuccessOrThrow(string url, TimeSpan timeout, CancellationToken cancellationToken = default)
     {
-        using var cts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
-        cts.CancelAfter(timeout);
+        using var timeoutCts = new CancellationTokenSource();
+        timeoutCts.CancelAfter(timeout);
```

Comment:
Can be made slightly simpler.

```suggestion
        using var timeoutCts = new CancellationTokenSource(timeout);
```